src/Stream/src/BinderRabbitMQ/Config/RabbitProducerOptions.cs

```diff
@@ -43,6 +43,8 @@ public RabbitProducerOptions()
 
         public string ConfirmAckChannel { get; set; }
 
+        public bool? UseConfirmHeader { get; set; }
```

Comment:
Does it make sense for this to be nullable? Why not just `bool`?

Reply:
I am just keeping all the options consistent